lib/OpenQA/Utils.pm

```diff
@@ -899,4 +900,6 @@ sub find_video_files { path(shift)->list_tree->grep(VIDEO_FILE_NAME_REGEX) }
 # workaround https://github.com/mojolicious/mojo/issues/1629
 sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/ }
 
+sub is_url { return shift =~ /http(s)?\:\/\//; }
```

Comment:
This checks for an URL only with the schemes http/https and an authority (even if empty). This makes sense here but maybe `is_http_url` would be a better name.